src/utils/tracking.ts

```diff
@@ -43,3 +44,41 @@ export const { Track, TrackAction, Identify, IdentifyAction } = setupTrackers<
   },
   { debug: process.env.NODE_ENV === 'development' },
 )
+
+interface Adt {
+  Tag: AdtTag
+}
+
+interface AdtTag {
+  t: number
+  c: string
+  tp: number
+  am: number
+  ti: string
+  xd: string
+  cpn?: string
+  doEvent: () => void
+}
+
+export const adtraction = (
+  orderValue: number,
+  orderId: string,
+  emailAddress: string,
+  couponCode: string | null,
+) => {
+  // @ts-ignore
```

Comment:
:( why?

Reply:
I guess I could declare my own types for it, that's true :-)